src/reference/config/bindings/binding-http-kafka.md

```diff
@@ -29,17 +29,17 @@ http_kafka_proxy:
           content-type: application/json
     - when:
         - method: GET
-          path: "/items/{id}"
+          path: /items/{id}
       exit: kafka_cache_client
       with:
         capability: fetch
         topic: items-snapshots
         filters:
           - key: "${params.id}"
     - when:
-        - path: "/items/{id}"
+        - path: /items/{id}
         - method: GET
-          path: "/items/{id};{correlationId}"
+          path: /items/{id};{correlationId}
```

Comment:
could this be `/items/{id}/{correlationId}` using a slash instead of a semicolon or maybe another URL encoding value. Does this support params like `/items/{id}?c={correlationId}`?

Reply:
It needs to line up with the `async` `location` below in `with`. I don't recall how much flexibility we have here.

Note that the async aspect of this is trying to return early with 202 accepted and then continue to receive the actual response via a subsequent GET request. So they are two parts of the same request-response, and logically belong at the same path, hence the use of a path parameter via `;` syntax for correlation id.